app/src/pages/Order/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { NavbarArrow } from '../../components/Navigation/NavbarArrow/NavbarArrow';
+import { OrderTable } from './components/OrderTable';
+import { useRecoilState } from 'recoil';
+import { basketState } from 'src/atoms';
+
+export const OrderPage = () => {
+  const [basketItems, setBaketItems] = useRecoilState(basketState);
+
+  const handleDeleteItem = (id: string) => {
+    console.log('delete item', id);
+
+    const newItems = basketItems.filter((item) => item.id !== id);
```

Comment:
Sensei @fmtabbara I dare to question where is var id coming from?? I may be too tired!

Reply:
this function gets passed in as a prop and is called by the child component. The child component has access to the basket item including id

Reply:
told you! I did not realised the function was inside handleDeleteItem!! :(